web/index.html

```diff
@@ -28,9 +28,11 @@
     <link href="packages/codemirror/addon/hint/show-hint.css" rel="stylesheet" media="screen">
     <link href="packages/codemirror/theme/zenburn.css" rel="stylesheet" media="screen">
     <link href="packages/dart_pad/editing/editor_codemirror.css" rel="stylesheet" media="screen">
-
     <script src="scripts/codemirror.js" defer></script>
-    <script type="application/dart" src="scripts/main.dart" defer></script>
+
+    <script defer src="scripts/main.dart" type="application/dart"></script>
+    <script defer src="packages/browser/dart.js"></script>
```

Comment:
Can you check, is this script reference properly removed in the `build/` dir output? (And, does the `scripts/main.dart` ref above keep the `defer` keyword in the re-written output?)